app/src/main/kotlin/net/syncthing/lite/library/DownloadFileTask.kt

```diff
@@ -4,15 +4,18 @@ import android.os.Handler
 import android.os.Looper
 import android.support.v4.os.CancellationSignal
 import android.util.Log
-import kotlinx.coroutines.experimental.launch
-import kotlinx.coroutines.experimental.suspendCancellableCoroutine
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.suspendCancellableCoroutine
 import net.syncthing.java.bep.BlockPullerStatus
 import net.syncthing.java.client.SyncthingClient
 import net.syncthing.java.core.beans.FileInfo
 import net.syncthing.lite.BuildConfig
 import org.apache.commons.io.FileUtils
 import java.io.File
 import java.io.IOException
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
```

Comment:
unused code?

Reply:
These are required now as resume and resumeWithException are not direct functions of the Continuation anymore but [extension functions](https://kotlinlang.org/docs/reference/extensions.html). resume is used at <https://github.com/syncthing/syncthing-lite/blob/0c6f947780401401ba02d88ce8ccc8c41df53e18/app/src/main/kotlin/net/syncthing/lite/library/DownloadFileTask.kt#L45> and resumeWithException at <https://github.com/syncthing/syncthing-lite/blob/0c6f947780401401ba02d88ce8ccc8c41df53e18/app/src/main/kotlin/net/syncthing/lite/library/DownloadFileTask.kt#L48>